Submodels/Legacy/testing_averaging.py

```
@@ -0,0 +1,69 @@
+import re
+import glob
+
+# Construct output file name
+output_file = f'averaged_file.test_case'
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)